include/deal.II/distributed/grid_tools.h

```diff
@@ -134,6 +134,60 @@ namespace parallel
     exchange_cell_data_to_ghosts (const MeshType &mesh,
                                   const std::function<boost::optional<DataType> (const typename MeshType::active_cell_iterator &)> &pack,
                                   const std::function<void (const typename MeshType::active_cell_iterator &, const DataType &)> &unpack);
+
+    /**
+     * A structure that allows the transfer of data of type T from one processor
+     * to another. It corresponds to a packed buffer that stores a list of
+     * cells and an array of type T.
+     *
+     * The vector @p data is the same size as @p cell_ids.
```

Comment:
You state it as a fact, but it's really a *requirement* that the user has to satisfy. Can you move this to the documentation of `save()` and mark it with `@pre` as a precondition for this function?